recipe/meta.yaml

```diff
@@ -28,6 +28,8 @@ requirements:
     - hdf5
 
 test:
+  requires:
+    - conda-build
```

Comment:
Why is `conda-build` required in the test environment? For the `conda inspect` commands below, `conda build` should be implicitly available via the root environment and should _not_ need to be installed in the package under test environment.

Reply:
On Prefect, I do not think `conda-build` is installed in the **root** environment but in an environment named **[prefect](https://github.com/anaconda-distribution/prefect-cookbook/blob/main/scripts/deploy_unix.sh#L19)**. Only Miniconda packages are in the root environment. 🤔

Reply:
Reverted this portion of the change.

Reply:
an error on osx-64 and osx-arm64:
`CommandNotFoundError: To use 'conda inspect', install conda-build.`

Reply:
That sounds like a mis-configuration of the macOS build workers (or possibly a bug in conda-build).

Reply:
interesting... looking into that tomorrow

Reply:
Remove essentially redundant `conda inspect` tests.